sessions/recap-project-6/example-implementation/package.json

```diff
@@ -30,7 +30,7 @@
     "jest": "^29.3.1",
     "jest-environment-jsdom": "^29.3.1"
   },
-  "description": "Recap Project 6: Tourio App - Example Implementation",
+  "description": "Recap Project 6: Tourio App",
```

Comment:
I would leave it how it was `"description": "Recap Project 6: Tourio App - Example Implementation",` in this case